main/default.cfg

```diff
@@ -7,45 +7,33 @@ unbindall
 //
 // WEAPONS/UPGRADES
 //
+
 bind  MOUSE1        +attack
 bind  MOUSE2        +attack2
 
-bind  [             weapprev
-bind  ]             weapnext
-bind  mwheelup      weapprev
-bind  mwheeldown    weapnext
+bind  mwheelup      "weapprev; +useitem"
+bind  mwheeldown    "weapnext; +useitem"
 bind  ENTER         +useitem
 bind  MOUSE3        +useitem
 bind  g             itemact grenade
 
+
 //
 // CHARACTER CONTROLS
 //
 
-bind  CTRL          +attack
-
-bind  ALT           +strafe
-bind  SHIFT         +speed
-bind  x             +sprint
-
-bind  PGDN          +lookdown
-bind  PGUP          +lookup
-bind  END           centerview
+bind  ALT         	+speed
+bind  SHIFT         +sprint
 
-bind  c             +movedown
+bind  CTRL          +movedown
 bind  SPACE         +moveup
 
-bind  UPARROW       +forward
-bind  DOWNARROW     +back
-bind  LEFTARROW     +left
-bind  RIGHTARROW    +right
+
```

Comment:
People who have played quake should be able to change their bindings accordingly. We do not want *new* players to use the arrow keys as it would be a great disadvantage (as there are very few keys nearby).